components/typography/metadata/typography-deprecated.yml

```diff
@@ -1,19 +1,181 @@
-id: heading-display
-name: Typography (Deprecated II)
+name: Typography (Deprecated)
+description: Deprecated typography components. The [new Typography components](typography.html) should be used instead.
 status: Deprecated
-details: These styles can be replicated using new Typography classes
-description: Legacy typography elements.
+hide: true
 examples:
-  - id: heading-display
+  - id: heading-1
     name: Standard
+    description: Typography elements paired to display clear content hierarchies.
     markup: |
-      <h1 class="spectrum-Heading--display">Display (h1)</h1>
-      <h2 class="spectrum-Heading--pageTitle">Page Title (h2)</h2>
-      <h2 class="spectrum-Heading--subtitle1">Subtitle 1 (h2)</h2>
-      <h3 class="spectrum-Heading--subtitle2">Subtitle 2 (h3)</h3>
-      <h4 class="spectrum-Heading--subtitle3">Subtitle 3 (h4)</h4>
-      <p class="spectrum-Body--small">Small Body Text</p>
-      <p class="spectrum-Body">Default Body Text</p>
-      <p class="spectrum-Body--secondary">Secondary Body Text</p>
-      <p class="spectrum-Body--italic"> Body Text Italic</p>
-      <p class="spectrum-Body--large">Large Body Text</p>
+      <div class="spectrum-Typography">
```

Comment:
isn't all this deprecated? Why bother updating the old one?

Reply:
I just renamed the files.